tests/functional/test_install_reqs.py

```diff
@@ -191,6 +191,42 @@ def test_install_local_editable_with_subdirectory(script):
     result.assert_installed('version-subpkg', sub_dir='version_subdir')
 
 
+def test_user_with_prefix_in_pydistutils_cfg(script, data, virtualenv):
+    """
```

Comment:
It may be just me, but aiee that docstring style hurts my brain. It is /permitted/ by the docstring pep but not needed.